protocol/build.rs

```diff
@@ -46,34 +46,16 @@ fn compile() {
     let out_dir = out_dir();
     fs::create_dir(&out_dir).expect("create_dir");
 
-    protobuf_codegen_pure::Codegen::new()
+    protobuf_codegen::Codegen::new()
+    .pure()
```

Comment:
Am I right that you could skip this line, making the code generator first try a binary `protoc` and fall-back to the pure Rust implementation?

Reply:
Unfortunately, the change is not that easy, e. g. there's no automatic pure fallback, but I'll think of something tomorrow, and same with the unwraps. Would you log a warning in a case of a failed enum deserialization, because that's majority of the unwraps now?

Reply:
Don't worry about the protobuf thing -- I remembered this was the case on some other project, when I forgot to add that `.pure()` call and it all happened automagically. It's not important, no need to add or change.

As for the unwraps: in the rest of librespot, it either fails from `try_from` or returns silently in other methods. If you think it's in some critical piece that warrants a log message, then we should consider making the entire method fallible with `Result`.